locales/bn/bn.json

```diff
@@ -429,26 +429,26 @@
     "Peru": "পেরু",
     "Philippines": "ফিলিপাইন",
     "Photo": "ছবি",
-    "Pitcairn": "মৃত্যুর স্থান",
+    "Pitcairn": "পিটকেয়ার্ন",
     "Please accept the terms of service.": "অনুগ্রহ করে সেবার শর্তসমূহ গ্রহণ করুন",
```

Comment:
Is the correct character used to end a sentence?

In other places, I noticed the use of the vertical slash character (`|`).

This translation was by machine.